scipy/stats/tests/test_multivariate.py

```diff
@@ -439,6 +588,14 @@ def test_multivariate_normal_rvs_zero_covariance(self):
         sample = model.rvs()
         assert_equal(sample, [0, 0])
 
+    def test_multivariate_normal_rvs_zero_inverse_covariance(self):
+        mean = np.zeros(2)
+        inverse_cov = np.zeros((2, 2))
+        model = multivariate_normal(
+            mean, cov=None, inverse_cov=inverse_cov, allow_singular=True)
+        sample = model.rvs()
+        assert_equal(sample, [0, 0])
```

Comment:
I don't think things need to work properly when the precision matrix is singular.